app/components/goals-list/goals-list.jsx

```diff
@@ -1,13 +1,28 @@
 import React from 'react';
 
+import GoalTile from '../goal-tile.jsx';
+
 const goalsList = ({goals}) => {
 
   const goalsListItems = goals.map(goal => {
-    return <li key={ goal.id }>{ goal.name }</li>
+    return <li
+      key={ goal.id }
+    >
+      <GoalTile
+        goal={ goal }
+      />
+    </li>
   });
 
+  const dynamicStyle = {
+    height: Math.max(window.innerHeight - 90, goals.length * 108 + 115),
+  };
+
   return (
-    <div className="page goalsList">
+    <div
+      className="page goalsList"
```

Comment:
Hmm, you need to choose a style. Some of your components you are putting props on new line, and some you are not... Does that make sense?